src/app/pages/common/entity/entity-form/validators/number-validation.ts

```diff
@@ -0,0 +1,26 @@
+import { ValidatorFn, FormControl } from '@angular/forms';
+
+export function numberValidator(): ValidatorFn {
+  let thisControl: FormControl;
+
+  return function numberValidate(control: FormControl) {
```

Comment:
Validator can probably be simplified.
I don't understand why we need `thisControl`.